release/pkg/tasks/release.go

```diff
@@ -13,10 +14,7 @@ import (
 
 // ReleaseNotes generates release notes for the current release.
 func ReleaseNotes(cfg *config.Config) {
-	outDir := cfg.RepoRootDir
-	if cfg.IsHashrelease {
-		outDir = cfg.HashreleaseDir()
-	}
+	outDir := filepath.Join(cfg.RepoRootDir, "release", "_output", "hashrelease")
```

Comment:
this also need to generate release notes for official release

Reply:
At the moment it's only used for hashreleases. I think we must have lost the hook into the release-notes target in the last PR.

Reply:
there's a release-notes target in root Makefile that calls release-notes target in release/Makefile and uses this function

Reply:
Yeah, I pushed a commit yesterday that fixed this - that make target was calling a CLI option that didn't exist any more.